src/fill-zulipdb-with-msgs-deploy.mjs

```diff
@@ -0,0 +1,191 @@
+// Reference to TypeScript definitions for IntelliSense in VSCode
+/// <reference path="../rnode-grpc-gen/js/rnode-grpc-js.d.ts" />
+
+import grpcLib from '@grpc/grpc-js';
+import { rhoParToJson } from '@tgrospic/rnode-grpc-js';
+import util from 'util';
+
+// RNode with environment parameters
+import { rnodeService } from './rnode-env.mjs';
+
+// Load .env file
+import { config } from 'dotenv';
+config();
+
+import postgres from 'postgres';
+
+/**
+  * @param {object} arg
+  * @param {typeof path} arg.path
+  */
+async function main(env, {grpcLib}) {
+  const zulip_db_config = {
+    host: 'localhost',
+    port: 5442,
+    database: 'zulip',
+    username: 'zulip',
+    password: process.env.POSTGRES_PASSWORD,
+  };
+  
+  const { DB_CONTRACT_URI } = env;
+
+  // Postgres connection
+  const sql = postgres(zulip_db_config);
```

Comment:
`postgres` instance is supplied to _main_ function but it's not defined as a field in the second parameter, so instance here is from import directly.

Reply:
Should I use the one from main or remove it as parameter?

Reply:
Preferably, _main_ should receive all dependencies without using _global_ scope.

@dckc has great collection about OCAP patterns and philosophy.
https://github.com/dckc/awesome-ocap